base/binaryplatforms.jl

```diff
@@ -597,6 +597,7 @@ const arch_mapping = Dict(
     "armv7l" => "arm(v7l)?", # if we just see `arm-linux-gnueabihf`, we assume it's `armv7l`
     "armv6l" => "armv6l",
     "powerpc64le" => "p(ower)?pc64le",
+    "riscv64" => "riscv64",
```

Comment:
Maybe
```suggestion
    "riscv64" => "(rv64|riscv64)",
```
as in https://github.com/alexfanqi/julia/commit/bfaed11d61b2eee02df92d2848f100a5fd65f149#diff-e6fe16743284c3f44ec75663f066e8022bd786bc41437f44ed9b5fe3f0ef22ab?  CC @alexfanqi, in case you're interested :eyes:

Reply:
What does LLVM call it?

Reply:
I updated my PR according to @alexfanqi's changes. They were quite similar, and their changes are different in a few places where I didn't have a clear picture of what to put.

Reply:
LLVM calls it `riscv64`.